Source/DafnyCore/Backends/Library/LibraryBackend.cs

```diff
@@ -75,7 +75,16 @@ public class LibraryBackend : ExecutableBackend {
     var dooPath = DooFilePath(dafnyProgramName);
 
     File.Delete(dooPath);
-    ZipFile.CreateFromDirectory(targetDirectory, dooPath);
+
+    try {
+      ZipFile.CreateFromDirectory(targetDirectory, dooPath);
+    } catch (IOException) {
+      if (File.Exists(dooPath)) {
```

Comment:
Nothing wrong with defensive coding and all, but why would the delete fail?

Reply:
I don't know, but it did.